### PR TITLE
Model.py TypeError

### PR DESCRIPTION
**Pull request by asvcode**
The requirements README state using Python 3.9 but this results in a `TypeError` when running` python -m scrapers.google_cloud_skill_boost.scrape_course_template`

```
Traceback (most recent call last):
  File "C:\Users\avird\anaconda3\envs\lpi\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\avird\anaconda3\envs\lpi\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\PillView\learning-path-index\app\course-scraper\src\scrapers\google_cloud_skill_boost\scrape_course_template.py", line 10, in <module>
    from scrapers.google_cloud_skill_boost.models import Activity, Course, CourseSubmodule
  File "C:\PillView\learning-path-index\app\course-scraper\src\scrapers\google_cloud_skill_boost\models.py", line 5, in <module>
    class Activity(BaseModel):
  File "C:\PillView\learning-path-index\app\course-scraper\src\scrapers\google_cloud_skill_boost\models.py", line 13, in Activity
    duration: int | float
TypeError: unsupported operand type(s) for |: 'type' and 'type'
```

This is because of:
`duration: int | float`

To make this work in either python 3.9 and in 3.10 and above, the following is used:

`duration: Union[int, float]`

## Summary by Sourcery

Fix a TypeError in models.py by updating type hints for compatibility with Python 3.9. Enhance error handling in scrape_journey.py to manage missing data more gracefully and update the script to prompt for user input. Update the README with additional Windows installation instructions.

Bug Fixes:
- Fix TypeError in models.py by replacing type hint 'int | float' with 'Union[int, float]' to ensure compatibility with Python 3.9 and above.

Enhancements:
- Improve error handling in scrape_journey.py by adding try-except blocks to handle missing elements gracefully when extracting journey details and links.
- Modify scrape_journey.py to prompt the user for the GCSB_JOURNEY_URL input and handle empty data scenarios before writing to a CSV file.

Documentation:
- Update README.md to include additional installation instructions for Windows, specifying the requirement for Microsoft Visual C++ 14.0 or greater.

### CODE REVIEW

**Review by TobeTek — COMMENTED** [review]
Great work @asvcode !
This PR seems to have the same changes as #82

**Review by neomatrix369 — CHANGES_REQUESTED** [review]
@asvcode hey have you looked through the comments from sourcery and made the changes? I'll re-review the work after you have applied them if you think one or more of them do not justify, then please put your comments there

If you need @TobeTek's help to check this out, then pls do ask about as well as he is closer to the code related to the scrapper

File: app/llm-poc-variant-01/README.md [hunk 1]
```diff
@@ -156,6 +160,14 @@ If this occurs then use the Docker container to run your commands, instructions
 
 #### via native shell CLI
 
+Before running ```lpiGPT.py```you need to specify the base URL for the Ollama API or the local instance of Ollama running on your machine. By default this will return a ```None``` value.
```
[the remark this answers is not in the document]
Comment (neomatrix369) [reply app/llm-poc-variant-01/README.md:163]: This one is a good one to apply

**Review by neomatrix369 — APPROVED** [review]
LGTM

**Pull request merged**